### ISSUE TITLE
RK3399 MJPEG dec + H.264 enc

### ISSUE THREAD

**Issue by coudy** [rockchip-linux/mpp#128]
Is it possible to run a live pipeline consisting of 1080p30 MJPEG HW decoding followed by H.264/5 HW encoding with mpp, ideally using the GStreamer plugins, on a RK3399 or a similar chip?

**Comment by HermanChen** [comment]
Yes, it is possible. But MJPEG decoding and H.264 encoding share the same VPU hardware. So the performance will drop.
The VPU hardware can support 1080p@30fps MJPEG decoding and H.264 encoding but no at the same time.
